apps/ramp-network/src/components/RampWidget.tsx

```diff
@@ -0,0 +1,43 @@
+import { useRef, useEffect } from 'react';
+import { RampInstantSDK, RampInstantEvent } from '@ramp-network/ramp-instant-sdk';
+
+type Props = {
+  url?: string;
+  assets: string;
+  address: string;
+  onClose?: () => void;
+};
+
+const WIDGET_CLOSE_EVENT = 'WIDGET_CLOSE';
+const PURCHASE_CREATED_EVENT = 'PURCHASE_CREATED';
+
+const RampWidget = ({ url, assets, address, onClose }: Props) => {
+  const containerNode = useRef(null);
+
+  useEffect(() => {
+    new RampInstantSDK({
+      url,
+      hostAppName: 'Ramp Network Safe App',
+      hostLogoUrl: 'https://docs.ramp.network/img/logo-1.svg',
+      swapAsset: assets,
+      userAddress: address,
+      containerNode: containerNode.current || undefined,
```

Comment:
we never set `containerNode`, is that correct?

Reply:
yep